helmfile/charts/prometheus-alerts/values.yaml

```
@@ -8,9 +8,15 @@ prometheusJob: kube-prometheus-stack-prometheus
 operatorJob: kube-prometheus-stack-operator
 prometheusNamespace: monitoring
 
-s3BucketPercentLimit: 80
-s3BucketSizeQuotaGB: 80
-
+s3BucketAlerts:
+  size:
+    enabled: false
+    percent: 80
+    sizeQuotaGB: 1000
+  nrOfObjects:
```

This is sufficient imo.
```suggestion
  objects:
```

---

You missed this one